alerting/provider/custom/custom.go

```diff
@@ -60,6 +60,8 @@ func (provider *AlertProvider) buildHTTPRequest(ep *endpoint.Endpoint, alert *al
 	url = strings.ReplaceAll(url, "[ENDPOINT_GROUP]", ep.Group)
 	body = strings.ReplaceAll(body, "[ENDPOINT_URL]", ep.URL)
 	url = strings.ReplaceAll(url, "[ENDPOINT_URL]", ep.URL)
+	body = strings.ReplaceAll(body, "[ENDPOINT_ERRORS]", strings.Join(result.Errors, "\n"))
+	url = strings.ReplaceAll(url, "[ENDPOINT_ERRORS]", strings.Join(result.Errors, "\n"))
```

Comment:
That does not feel like the right separator to use to explode an array into a URL. 
I think maybe we'd be better off just using a comma for both the body and the URL for now